recipes-soc/rdk-gstreamer-utils-platform/rdk-gstreamer-utils-platform.bb

```diff
@@ -0,0 +1,31 @@
+LICENSE = "CLOSED"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS += "${@bb.utils.contains('DISTRO_FEATURES', 'gstreamer1', 'gstreamer1.0  gstreamer1.0-plugins-base', 'gstreamer gst-plugins-base', d)}"
+DEPENDS += "rdk-gstreamer-utils-headers"
+
+SRC_URI = "git://git@github.com/ssadha725_comcast/rdk-gstreamer-utils-raspberrypi;${RDKE_GITHUB_SRC_URI_SUFFIX}"
```

Comment:
Create a CMFSUPPORT ticket to create a new repository for rdk-gstreamer-utils-raspberrypi in github/rdkcentral.

Reply:
Done